mobify/platform-scaffold
# [Integration manager] merlins checkout payment page

This PR moves merlin's specific checkout payment actions to the merlin's connector. 

**Note:** there is a bug where you can't continue past the shipping page if you log in on that page, the bug is unrelated to these changes and will be fixed in a separate PR

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-47
 **Linked PRs**: n/a

## Changes
- Added fetcher for checkout payment container
- Removed CheckoutPayment section of app/actions global fetch 
- Moved page fetch and form submission actions to merlins connector checkout commands file
- Fixed Checkout Payment action tests
- Updated template.jsx fetcher call to use fetchURL when it's defined

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add an item to your cart and proceed through the checkout
- You should be able to complete an order
